Controllers/AdminController.js

```diff
@@ -0,0 +1,177 @@
+const AsyncHandler = require("express-async-handler");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const Admin = require("../Models/AdminModel");
+const Book = require("../Models/BookModel");
+const Student = require("../Models/StudentModel");
+const DefinedError = require("../Middleware/DefinedError");
+const { errHandle } = require("../Middleware/errHandle");
+
+const RegisterUser = async (req, res) => {
+  try {
+    const { username, password, name } = req.body;
+
+    if (!username || !password || !name) {
```

Comment:
```suggestion
    if ([username, password, name].some(field => !field)) {
```
Check Array.prototype.some docs